### PR TITLE
add fall 2020 event cards and highlights

### PR DESCRIPTION
**Pull request by jodymlin**
* new blurbs and banners added for Fall 2020 events
* will add FB event links as the dates get closer and the directors create them
* also didn't want to delete all old event highlight descriptions in case we wanted to use them in the future, so I created an `archived-highlights.js` file to keep these for now

### CODE REVIEW

File: src/data/events/highlights.js [hunk 1]
```diff
@@ -1,78 +1,47 @@
...
-	{
-		name: 'Passion Talks',
-		description: passiontalksDescription,
-		imgFilePath: 'event/2020s-passiontalks.png',
-		button: 'Recordings',
-		link: 'https://www.youtube.com/watch?v=BB9uzqJgeMg&list=PLPO7_kXilXFartXh7vu7OI6zq8JOF5k5u&index=2'
-
+		name: 'Bloom: Hackschool',
+		description: bloomDescription,
+		imgFilePath: 'event/2020f-bloom-hackschool-banner.png', // TODO: change
```
Comment (jodymlin) [inline src/data/events/highlights.js:29]: Ignore this `TODO` . I forgot to remove it before I pushed. 

I will remove it in the next PR to update events. I don't wanna waste extra netlify build time when just to remove this comment in a new commit.

Comment (GalenWong) [reply src/data/events/highlights.js:29]: we are not that tight on netlify build budget haha

File: src/data/events/archived-highlights.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+	},
+	{
+		name: 'Tooling Series',
+		description: toolingseriesDescription,
+		imgFilePath: 'event/2020s-toolingseries.png',
+		button: 'Curriculum',
+		link: 'https://github.com/uclaacm/hack-tooling-series'
+	}
+];
```
Comment (GalenWong) [inline src/data/events/archived-highlights.js:80]: Technically all these history would be retained by git so we don't have to fear losing them. I am not too sure about keeping stuff in a code file. What does @TimothyGu think about this?

Comment (GalenWong) [inline src/data/events/archived-highlights.js:80]: Technically all these history would be retained by git so we don't have to fear losing them. I am not too sure about keeping stuff in a code file. What does @TimothyGu think about this?

File: src/components/EventCard/EventCard.js [hunk 3]
```diff
@@ -119,7 +119,7 @@ function EventCard({
 							{name}
 						</Typography>
 						<Typography variant="body1" component="span" classes={{ root: classes.details }}>
-							{`${dateStr} · ${location}`}
+							{`${dateStr}`}{ location ? ` · ${location}` : null }
```
Comment (TimothyGu) [inline src/components/EventCard/EventCard.js:122]:
```suggestion
							{dateStr}{ location ? ` · ${location}` : null }
```

File: src/data/events/archived-highlights.js (hunk 2 above)
Comment (TimothyGu) [reply src/data/events/archived-highlights.js:80]: +1. Let's not intentionally put dead code in the repo.

**Review by GalenWong — APPROVED** [review]

**Review by TimothyGu — APPROVED** [review]

**Pull request merged**